### PR TITLE
Reformatting TensorFlow and AnomalyDetection samples to width 85

### PR DESCRIPTION
**Pull request by sayanshaw24**
Guidelines followed:
- 85 characters per line
- Use 4 spaces for indentation
- Dot, open parentheses, and function/variable name on same line
- Math operations on same line
- Don't indent comments
- Don't break links
- Don't break a comment if it represents a print output
- Add an extra line after a break if it does not already exist
- Break before "$"

Fix for Issue #3478

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 1]
```diff
@@ -13,27 +13,35 @@ public static class TextClassification
...
-                "remarks throughout the film were great it was just brilliant so much that i bought the " +
-                "film as soon as it was released for  and would recommend it to everyone to watch and the " +
-                "fly fishing was amazing really cried at the end it was so sad and you know what they say " +
-                "if you cry at a film it must have been good and this definitely was also  to the two " +
-                "little boy's that played the  of norman and paul they were just brilliant children are " +
-                "often left out of the  list i think because the stars that play them all grown up are " +
-                "such a big profile for the whole film but these children are amazing and should be praised " +
-                "for what they have done don't you think the whole story was so lovely because it was true " +
-                "and was someone's life after all that was shared with us all" } };
+                Sentiment_Text = "this film was just brilliant casting location" +
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Maybe we need to do
```suggestion
                Sentiment_Text = "this film was just brilliant casting location " +
```
for all new lines.

Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Yes, I will change that and submit a new PR.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 2]
```diff
@@ -43,49 +51,68 @@ public static void Example()
                );
 
             // Load the TensorFlow model once.
-            //      - Use it for quering the schema for input and output in the model
+            //      - Use it for quering the schema for input and output in the
+            //            model
             //      - Use it for prediction in the pipeline.
-            var tensorFlowModel = mlContext.Model.LoadTensorFlowModel(modelLocation);
+            var tensorFlowModel = mlContext.Model.LoadTensorFlowModel
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Not sure if it should be
```
LoadTensorFlowModel(
    modelLocation)
```

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 3]
```diff
@@ -43,49 +51,68 @@ public static void Example()
...
-            var predictionType = (VectorDataViewType)schema["Prediction/Softmax"].Type;
-            Console.WriteLine("Name: {0}, Type: {1}, Shape: (-1, {2})", "Prediction/Softmax", predictionType.ItemType.RawType, predictionType.Dimensions[0]);
-            
-            // The model expects the input feature vector to be a fixed length vector.
-            // In this sample, CustomMappingEstimator is used to resize variable length vector to fixed length vector.
+            Console.WriteLine("Name: {0}, Type: {1}, Shape: (-1, {2})", "Features",
+                featuresType.ItemType.RawType, featuresType.Dimensions[0]);
+
+            var predictionType = (VectorDataViewType)schema["Prediction/Softmax"]
+                .Type;
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Is it `.Type` or `Type`?

Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: It is .Type

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 4]
```diff
@@ -105,7 +132,8 @@ public class IMDBSentiment
 
             /// <summary>
             /// This is a variable length vector designated by VectorType attribute.
-            /// Variable length vectors are produced by applying operations such as 'TokenizeWords' on strings
+            /// Variable length vectors are produced by applying operations such as
+            /// 'TokenizeWords' on strings
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: This line has some space for storing tokens in the line right below.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs [hunk 5]
```diff
@@ -59,12 +64,14 @@ public static void Example()
...
+                    Console.WriteLine("The {0}-th example with features [{1}] is" +
+                        "an inlier with a score of being inlier {2}", i, 
+                        featuresInText, result.Score);
                 else
                     // The i-th sample is predicted as an outlier.
-                    Console.WriteLine("The {0}-th example with features [{1}] is an outlier with a score of being inlier {2}",
-                        i, featuresInText, result.Score);
+                    Console.WriteLine("The {0}-th example with features [{1}] is" +
+                        "an outlier with a score of being inlier {2}",
+                            i, featuresInText, result.Score);
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs]:
```suggestion
                        i, featuresInText, result.Score);
```

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs [hunk 6]
```diff
@@ -52,12 +58,14 @@ public static void Example()
 
                 if (result.PredictedLabel)
                     // The i-th sample is predicted as an inlier.
-                    Console.WriteLine("The {0}-th example with features [{1}] is an inlier with a score of being inlier {2}",
-                        i, featuresInText, result.Score);
+                    Console.WriteLine("The {0}-th example with features [{1}]" +
+                        "is an inlier with a score of being inlier {2}", i,
+                            featuresInText, result.Score);
```
Comment (wschin) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs]:
```suggestion
                        featuresInText, result.Score);
```
Would you think this change makes things better?

Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSample.cs]: I'm not sure. I was looking at the Clustering formatting changes @artidoro made and I think he indented after every break, other than comments, but I'll find out and change it.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs (hunk 2 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Yep, changed that on the most recent commit.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs (hunk 4 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: Changed on new commit

File: docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs (hunk 5 above)
Comment (sayanshaw24) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/AnomalyDetection/RandomizedPcaSampleWithOptions.cs]: Changed in new commit.

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 7]
```diff
@@ -43,49 +51,68 @@ public static void Example()
...
+            //         dictionary ('lookupMap'),
+            //      3. Resizes the integer vector to a fixed length vector using
+            //         CustomMappingEstimator ('ResizeFeaturesAction')
             //      4. Passes the data to TensorFlow for scoring.
-            //      5. Retreives the 'Prediction' from TensorFlow and put it into ML.NET Pipeline 
+            //      5. Retreives the 'Prediction' from TensorFlow and put it into
+            //         ML.NET Pipeline 
 
-            Action<IMDBSentiment, IntermediateFeatures> ResizeFeaturesAction = (i, j) =>
+            Action<IMDBSentiment, IntermediateFeatures> ResizeFeaturesAction = (i,
```
Comment (natke) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: This would be clearer if the (i, j) was on the same line

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs [hunk 8]
```diff
@@ -43,49 +51,68 @@ public static void Example()
...
 
-            var model = mlContext.Transforms.Text.TokenizeIntoWords("TokenizedWords", "Sentiment_Text")
-                .Append(mlContext.Transforms.Conversion.MapValue("VariableLengthFeatures", lookupMap,
-                    lookupMap.Schema["Words"], lookupMap.Schema["Ids"], "TokenizedWords"))
-                .Append(mlContext.Transforms.CustomMapping(ResizeFeaturesAction, "Resize"))
-                .Append(tensorFlowModel.ScoreTensorFlowModel("Prediction/Softmax", "Features"))
-                .Append(mlContext.Transforms.CopyColumns("Prediction", "Prediction/Softmax"))
-                .Fit(dataView);
-            var engine = mlContext.Model.CreatePredictionEngine<IMDBSentiment, OutputScores>(model);
+            var model = mlContext.Transforms.Text.TokenizeIntoWords(
```
Comment (natke) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs]: It would be better to try and align the Append methods, as well as keep the transform strings together. Something like:

        var model =
        mlContext.Transforms.Text.TokenizeIntoWords(
            "TokenizedWords",
            "Sentiment_Text")
        .Append(mlContext.Transforms.Conversion.MapValue(
            "VariableLengthFeatures",
            lookupMap,
            lookupMap.Schema["Words"],
            lookupMap.Schema["Ids"],
            "TokenizedWords"))
        .Append(mlContext.Transforms.CustomMapping(
            ResizeFeaturesAction,
            "Resize"))
        .Append(tensorFlowModel.ScoreTensorFlowModel(
            "Prediction/Softmax",
            "Features"))
        .Append(mlContext.Transforms.CopyColumns(
            "Prediction",
            "Prediction/Softmax"))
        .Fit(dataView);

File: docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs [hunk 9]
```diff
@@ -86,9 +93,12 @@ public class TensorData
         public static TensorData[] GetTensorData()
         {
             // This can be any numerical data. Assume image pixel values.
-            var image1 = Enumerable.Range(0, inputSize).Select(x => (float)x / inputSize).ToArray();
-            var image2 = Enumerable.Range(0, inputSize).Select(x => (float)(x + 10000) / inputSize).ToArray();
-            return new TensorData[] { new TensorData() { input = image1 }, new TensorData() { input = image2 } };
+            var image1 = Enumerable.Range(0, inputSize).Select(
+                x => (float)x / inputSize).ToArray();
+            var image2 = Enumerable.Range(0, inputSize).Select(
+                x => (float)(x + 10000) / inputSize).ToArray();
```
Comment (codemzs) [inline docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/ImageClassification.cs]: As discussed new line between any two lines that were broken into multiple lines.
